pkg/webhooks/github/actions/common.go

```diff
@@ -186,23 +197,23 @@ func (w *WebhookActions) ProcessReleaseEvent(ctx context.Context, payload *ghweb
 	}
 }
 
-func (w *WebhookActions) ProcessPullRequestEvent(ctx context.Context, payload *ghwebhooks.PullRequestPayload) {
+func (w *WebhookActions) ProcessPullRequestEvent(ctx context.Context, payload *github.PullRequestEvent) {
 	ctx, cancel := context.WithTimeout(ctx, constants.WebhookHandleTimeout)
 	defer cancel()
 	g, _ := errgroup.WithContext(ctx)
 
 	for _, a := range w.dp {
 		g.Go(func() error {
-			if payload.Action != "opened" || payload.Repository.Name != "docs" {
+			if pointer.SafeDeref(payload.Action) != "opened" || pointer.SafeDeref(payload.Repo.Name) != "docs" {
```

Comment:
Are there constants for `"opened"`? Should we add one?

Reply:
At least in the go-github repository there are no constants for them. They are all just hidden in the godocs, e.g.: https://github.com/google/go-github/blob/ac631dc3f83dcbfb6ffaffd8e7e28c59f2ad3264/github/event_types.go#L749

Same question if this should go into a refactoring PR in order not to go too far from the original purpose of this PR.